setup.py

```diff
@@ -58,5 +59,8 @@ def run(cls):
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: Implementation :: PyPy",
         "Topic :: Software Development :: Libraries :: Python Modules",
+    ],
+    install_requires=[
+        "sortedcontainers==2.1.0"
```

Comment:
kafka-python has to-date not taken on any external dependencies, and we'd like to keep with that philosophy. Is this module necessary / can we refactor to use the standard library instead? If it is necessary, can we make it optional and disable the assignor if it is not installed?

Reply:
It'd be really nice to see this land without an external dependency...

Reply:
Yep, thought about it as well, should not be a rocket science to implement sorted sets by hand. Will do.